data/scripts/creaturescripts/others/remove_empty_parcel.lua

```diff
@@ -1,18 +1,25 @@
 local removeEmptyParcelsEvent = CreatureEvent("RemoveEmptyParcelsOnLogin")
 
 function removeEmptyParcelsEvent.onLogin(player)
-	local emptyParcelsToRemove = {}
-	for _, parcel in ipairs(player:getStoreInbox():getItems(true)) do
-		if parcel:getId() == ITEM_PARCEL_STAMPED and parcel:getEmptySlots() == 10 then
-			table.insert(emptyParcelsToRemove, parcel)
+	local inbox = player:getInbox()
```

Comment:
Yes, this script was created when buying items in the store creates a parcel in the store inbox. I've already talked with Marcos.

Reply:
Actually, you only mentioned deleting from the store inbox, which doesn't make sense because the inbox is different from the store inbox. Parcels received go to the inbox, as well as items from the market.
The script is deleting parcels sent by other players, and an error is being returned when it's unable to retrieve them for some reason.
The verification is incorrect, and the script is faulty either way.